spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/TraceScope.java

```diff
@@ -0,0 +1,86 @@
+package org.springframework.cloud.sleuth;
+
+import java.io.Closeable;
+
+import lombok.SneakyThrows;
+import lombok.Value;
+import lombok.experimental.NonFinal;
```

Comment:
I'm not worried about it.  Lombok doesn't "ship" with spring cloud it's just compile time.